packages/components/organisms/f-checkout/src/services/mapper.js

```diff
@@ -0,0 +1,40 @@
+const mapCheckoutRequest = ({
+    address,
+    customer = {},
+    isCheckoutMethodDelivery,
+    time,
+    userNote
+}) => ({
+    customer: {
+        firstName: customer.firstName || null,
+        lastName: customer.lastName || null,
+        phoneNumber: customer.mobileNumber || null,
+        dateOfBirth: null
+    },
+    fulfilment: {
+        time,
+        location: {
+            ...(isCheckoutMethodDelivery ? {
+                address: {
+                    lines: [
+                        address.line1 || '',
+                        address.line2 || '',
+                        '',
+                        address.city || '',
+                        ''
+                    ],
+                    postalCode: address.postcode || null
+                }
+            } : {}),
+            geolocation: null
+        }
+    },
+    notes: [
+        {
+            type: 'delivery',
```

Comment:
Is this correct, or should it sometimes be `'collection'`?

Reply:
I checked with Dave on Slack and he said
> "There is no need to differentiate between collection and delivery. These are delivery notes. For now, we will show them for both delivery and collection to not cause any regressions with what is already there."